producer.py

```diff
@@ -120,34 +124,36 @@ async def main() -> None:
     si_unanalyzed_package = SIUnanalyzedPackageMessage()
 
     for package_name, package_version, index_url, solver_name in packages:
-        for index_url in [index_url] if index_url is not None else indexes:
-            try:
-                async_tasks.append(
-                    unresolved_package.publish_to_topic(
-                        unresolved_package.MessageContents(
-                            package_name=package_name,
-                            package_version=package_version,
-                            index_url=[index_url],
-                            solver=solver_name,
-                            component_name=COMPONENT_NAME,
-                            service_version=__service_version__,
+        if THOTH_GRAPH_REFRESH_SOLVER:
```

Comment:
What about adding a check at the beginning, if all three env variables are set to 0 just return doing nothing and before some logging saying `all variables are set to 0, no messages of any type will be sent` or similar.

Reply:
Also useful to have a logging saying what is actually set to 1, so we know what messages we expect, WDYT?

Reply:
Sure!